css/style.css

```diff
@@ -771,6 +771,9 @@ Main Components
   .the-venue .the-venue-detailts h2{
     font-size: 30px;
   }
+  .get-register div {
```

Comment:
This has been added b/c `.cd-section > div` (line 55) sets a value of 100vh and this contributes to the problems.